deployments/deploy-tests.ps1

```diff
@@ -0,0 +1,145 @@
+try{
+    Import-Module .\azure\az-cloud-context\az-cloud-context.psd1
+    Import-Module .\azure\resources\key-vault\key-vault.psd1
+    Import-Module .\azure\resources\log-analytics\log-analytics.psd1
+    Import-Module .\powerform\pf-deployment-context\pf-deployment-context.psd1
+
+    Get-Command -Module log-analytics
+    Get-Command -Module key-vault
+    Get-Command -Module pf-deployment-context
+    Get-Command -Module az-cloud-context
+
+    ##############################################################################################################
+    #
+    #               Deployment Context can drive naming defaults 
+    #
+    #############################################################################################################
+
+    #Still need to build this. The thought behind this is that a user wants to deploy resources out with best practices/enterprise standards
+    #baked into the module.  By defining the deploy context parameters, the names will be created, therefore it knows
+    #the RG, Sub, Resources names.  All options are going to be the defaults, so just build the definitions and deploy
+
+    #$deployContext = New-PfDeploymentContext -CompanyAbbreviation "MYAEA" -GroupAbbreviation "KKZH" -TeamAbbreviation "ADM" -EnvironmentLetter "D"
+
+    #name, az context built from deployment context params above
+    #$kv = New-KeyVaultDefinition
+
+    #name, az context built from deployment context params above
+    #$la = New-LogAnalyticsDefinition
+
+    #$deployContext.ResourceDefinitions.AddRange(@($kv,$la))
+    #$deployContext.Deploy()
+
+
+    ##############################################################################################################
+    #
+    #               Deploy multiple resources with context and deployment context
+    #
+    #############################################################################################################
+
+    #Still need to write the deploy method, but the thought here is that you get the sub and rg from the
+    #current context so all you need to do is define the name and other options
+
+    $deployContext = New-PfDeploymentContext
+
+    Set-AzCloudContext -SubscriptionName "MYAEA-KKZH-D" -ResourceGroupName "RG-MYAEA-KKZH-ADM-D"
+
```

Comment:
after seeing the PfDeploymentContext name, wonder if we should like Pf-prefix all our stuff to distinguish between our stuff and Az, since sometimes we also using Az name

Reply:
I thought about that.  But now you are going to have PfAz in front of all our methods/functions.  I am ok with that if you are.  That would get us back to something like "Set-PfAzContext"  and "New-PfAzKeyVault"  Thoughts?